cli/roger_push.py

```diff
@@ -215,22 +224,32 @@ def main(self, settings, appConfig, frameworkObject, hooksObj, args):
                         environment = env_var
             else:
                 environment = args.env
+            # ----------------------------------------------
 
             if environment not in roger_env['environments']:
                 raise ValueError("Environment not found in roger-mesos-tools.config file.")
 
+            # ----------------------------------------------
+            # GetEnvironmentConfig(environment)
+            # ----------------------------------------------
             environmentObj = roger_env['environments'][environment]
             common_repo = config.get('repo', '')
+
+            # ----------------------------------------------
+            # GetContainersForApp(app)
+            # ----------------------------------------------
             app_name = args.app_name
             container_list = []
+            # todo (vmahedia): What does ':' signify? Put explanation.
```

Comment:
The : and the , are used when deploying multiple. The : is supposed to signify multiple containers under a common application name ala feedstore, https://marathon-prod-roger.dal.moz.com/ui/#/group/%2Fmoz-kwe%2Ffeedstore. The , is supposed to be multiple different applications I believe.